bin/current-account-config-name.sh

```diff
@@ -4,4 +4,8 @@
 # infra/accounts that matches "<account name>.<account id>.s3.tfbackend".
 # The config name is "<account name>.<account id>""
 set -euo pipefail
-ls -1 infra/accounts | grep "$(./bin/current-account-id.sh)" | grep s3.tfbackend | sed 's/.s3.tfbackend//'
+CURRENT_ACCOUNT_ID="$(./bin/current-account-id.sh)"
+BACKEND_CONFIG_FILE_PATH="$(ls -1 infra/accounts/*."$CURRENT_ACCOUNT_ID".s3.tfbackend)"
+BACKEND_CONFIG_FILE="$(basename "$BACKEND_CONFIG_FILE_PATH")"
+BACKEND_CONFIG_NAME="${BACKEND_CONFIG_FILE/.s3.tfbackend/}"
+echo "$BACKEND_CONFIG_NAME"
```

Comment:
the shell check linter suggested these changes.

i tested that this still works by running this script from the platform-test account:

<img width="492" alt="image" src="https://github.com/navapbc/template-infra/assets/447859/ae23eafc-d79b-46d0-b42b-d8a863dbaeec">